documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,423 @@
+      },
+      {
+        "type": "cryptowatch",
+        "exchange": "binance"
+      },
+      {
+        "type": "cryptowatch",
+        "exchange": "bitstamp"
+      }
+    ]
+  },
+}
+  ... Rest of bot startup logs and continuous health reports...
+```
+
+The bots are now running! Any liquidation event or informative logs will be printed here.
+If all is operating correctly, you should be able to liquidate incorrectly collateralized positions and dispute incorrectly liquidated positions.
+
+## Running the bots locally with Docker
+
+Running the bots from your local machine is simple but is not ideal for a long term bot execution as you'll need to keep a terminal running the whole time to keep the bot process alive.
+What would be better is to run the bots within an isolated Docker container that can run in the background on your machine.
+
+The steps followed here can also be re-produced on a VPS service like Google Cloud Compute Engine, Digital Ocean or AWS EC2 to host your Dockerized bots in the cloud.
+Section 3 of this tutorial will show you how to deploy the bots to GCP. It is recommended that you get the bots running within a local Docker environment before trying to run them on the cloud.
+
+**a) Setting docker up on your local machine**
+
+To be able to run the bots within a Dockerized environment you need to have Docker set up on your local machine. Getting started with Docker is relatively straight forward.
+See the official docks for [Mac](https://docs.docker.com/docker-for-mac/), [Linux](https://docs.docker.com/engine/install/ubuntu/) or [Windows](https://docs.docker.com/docker-for-windows/install/).
+
+**b) Creating bot environment configurations**
+
+In the previous section, both the liquidator and disputer bots used the same `.env` configuration file.
+In this section, we will create separate configuration files for each bot.
+These scripts will contain all the settings for a given bot, as well as the starting command used to boot the bot.
+
+Start by copying the `.env` you created to make two new env files. This section assumes you are in the `/core` directory. Run the following commands:
+
+```bash
+# Copy the contents of the .env and add command to run the liquidator bot.
+cp .env liquidator.env
+echo '\nCOMMAND=npx truffle exec ../liquidator/index.js --network kovan_mnemonic' >> liquidator.env
+
+# Do the same for the disputer bots
+cp .env disputer.env
+echo '\nCOMMAND=npx truffle exec ../disputer/index.js --network kovan_mnemonic' >> disputer.env
+```
+
+You should now have two config files `liquidator.env` and `disputer.env` within the `/core` directory which contain the original configs defined in the previous section along with a `COMMAND` line which defines the execution command of the bot.
+These commands are the same as before.
+
+**c) Starting the Docker containers**
+
+Next, we will start the Docker containers in detached mode on our local machine. To do this run the following:
+
```

Comment:
nit: just want to double check, does `docker run` automatically pull the latest images? Or do you have to run `docker pull` first to get the base images on your local machine?

Reply:
bump

Reply:
so if the user has not run the tutorial before then they will get the latest. if they have then they might get a stale image. I'll update to get the user to do both.